app/questionnaire/questionnaire_schema.py

```diff
@@ -83,6 +84,28 @@ def get_answers_by_answer_id(self, answer_id):
         """
         return self._answers_by_id.get(answer_id)
 
+    def get_default_answer(self, answer_id):
+        try:
+            block = self.get_block_for_answer_id(answer_id)
+        except TypeError:
+            return None
+
+        try:
+            question = block['question']
+        except KeyError:
+            question = block['question_variants'][0]['question']
+
+        answers = [
+            answer for answer in question['answers'] if answer['id'] == answer_id
+        ]
```

Comment:
This should be a single answer rather than an array. Answer ids in a given question are unique, so this would only ever generate a list of 0 or 1 items.